packages/interface/src/features/ballot/components/BallotConfirmation.tsx

```diff
@@ -100,12 +111,60 @@ export const BallotConfirmation = ({ pollId }: IBallotConfirmationProps): JSX.El
 
         <Notice
           content={
-            round?.votingEndsAt ? format(round.votingEndsAt, "d MMM yyyy hh:mm") : "The date would be announced soon."
+            round?.votingEndsAt
+              ? `Results will be available after tallying - ${format(round.votingEndsAt, "d MMM yyyy hh:mm")}`
+              : "The date would be announced soon."
           }
-          title="Results will be available after tallying."
-          variant="block"
         />
 
+        <Card variant="invert">
+          <div className="text-center">
+            <b className="text-2xl uppercase">Share that you have voted in {round?.roundId} round.</b>
```

Comment:
will this not read like Share that you have voted in 1 round? or 2 round, etc.? maybe just "in the round"?

Reply:
Not quite sure I understand what you said, so it's weird to share the amount of user voted since this `/round/[pollId]/ballot/confirmation` page is right after user submit their ballot in a round. And if we remove the `{round?.roundId}` part and make this sentence a constant, would it be weird for those who read this post?

Reply:
round?.roundId will be something like 0, 1 or 2 (poll Id) right? or is it now the round name?

Reply:
roundId is the name of the roundId, and pollId, which is used on the url page redirection, is 0, 1, 2, etc.

Reply:
ohhh nice ok my bad